src/blockchain/transactions.ts

```diff
@@ -103,6 +103,38 @@ export type TxState = {
 
 let txCounter: number = 1;
 
+type NodeCallback<I, R> = (
+  i: I,
```

Comment:
What does this `i` should represent?

Reply:
I see it depends on the type provided but still it represents something

Reply:
type of function that fits node callback definition